src/compounds/product-table/src/components/rowWrapper.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+  link?: string
+  children?: React.ReactNode
+}
+
+const linkWrapper = (link: string, children: React.ReactNode) => {
+  return (
+    <a
+      sx={{
+        textDecoration: 'none',
+        color: 'inherit',
+        variant: 'compounds.product-table.row.linkWrapper'
+      }}
+      href={link}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <div
```

Comment:
could you put this padding up onto the linkWrapper?

Reply:
If I move it into the anchor, it doesn't apply the padding correctly 
![image](https://user-images.githubusercontent.com/56200818/92090568-7971d700-edc7-11ea-97a6-ed7b3970ba94.png)

Reply:
you probably need to add display: block to the anchor, anchors by default are inline so cant have padding

Reply:
of course! just made the change